rpc/estimate_fee.go

```diff
@@ -37,10 +37,45 @@ type FeeEstimate struct {
 	Unit            *FeeUnit   `json:"unit,omitempty"`
 }
 
+type FeeEstimate struct {
+	L1GasConsumed     *felt.Felt `json:"l1_gas_consumed,omitempty"`
+	L1GasPrice        *felt.Felt `json:"l1_gas_price,omitempty"`
+	L2GasConsumed     *felt.Felt `json:"l2_gas_consumed,omitempty"`
+	L2GasPrice        *felt.Felt `json:"l2_gas_price,omitempty"`
+	L1DataGasConsumed *felt.Felt `json:"l1_data_gas_consumed,omitempty"`
+	L1DataGasPrice    *felt.Felt `json:"l1_data_gas_price,omitempty"`
+	OverallFee        *felt.Felt `json:"overall_fee"`
+	Unit              *FeeUnit   `json:"unit,omitempty"`
+}
+
 /****************************************************
 		Estimate Fee Handlers
 *****************************************************/
 
+func FeeEstimateToV0_7(feeEstimate FeeEstimate) FeeEstimateV0_7 {
```

Comment:
Do you use this function outside of this package? If not please make it private

Reply:
[Make feeEstimateToV0_7 private](https://github.com/NethermindEth/juno/pull/2335/commits/1d1d2a928a3c928a21c39609cf105f6ddee9a880)
[Make feeEstimateToV0_7 private](https://github.com/NethermindEth/juno/pull/2335/commits/5faca913f3b47db986781aff0c2277e4d0c06714)